src/main/java/com/minecolonies/coremod/commands/colonycommands/CommandShowClaim.java

```diff
@@ -36,10 +36,10 @@ public int onExecute(final CommandContext<CommandSourceStack> context)
         final ServerLevel level = context.getSource().getLevel();
 
         // Colony
-        BlockPos pos = context.getSource().getEntity().blockPosition();
+        BlockPos pos = BlockPos.containing(context.getSource().getPosition());
```

Comment:
can just get blockPosition from an entity too

Reply:
the source is better as it allows the command to run from commandline^^